### PR TITLE
More fixes for #568 - Open Url in browser

### PR DESCRIPTION
**Pull request by Dean151**
I saw the fix for StatusRowContextMenu, so this is a complement for this same bug accros the app

This is related with the changes we made to the router to "Open in Ice Cubes" 😬🫣

### CODE REVIEW

File: Packages/Status/Sources/Status/Row/StatusCardView.swift [hunk 1]
```diff
@@ -64,7 +64,7 @@ public struct StatusCardView: View {
         ShareLink(item: url) {
           Label("Share this link", systemImage: "square.and.arrow.up")
         }
-        Button { openURL(url) } label: {
+        Button { UIApplication.shared.open(url) } label: {
```
Comment (Dimillian) [inline Packages/Status/Sources/Status/Row/StatusCardView.swift:67]: But for the card, you still want to use the routeur right? It should respect the open in app browser vs open in system browser.

File: Packages/Status/Sources/Status/Row/StatusRowContextMenu.swift [hunk 2]
```diff
@@ -8,8 +8,6 @@ struct StatusRowContextMenu: View {
   @EnvironmentObject private var currentInstance: CurrentInstance
   @EnvironmentObject private var routerPath: RouterPath
 
-  @Environment(\.openURL) var openURL
```
Comment (Dimillian) [inline Packages/Status/Sources/Status/Row/StatusRowContextMenu.swift:11]: good cleanup.

File: Packages/Status/Sources/Status/Row/StatusCardView.swift (hunk 1 above)
Comment (Dean151) [reply Packages/Status/Sources/Status/Row/StatusCardView.swift:67]: Not sure where the card is located, I was following the "Open in browser" translation.
Problem is that currently, openURL called on a Mastodon url somehow always end up to be an in-app trigger from my understanding

Comment (Dimillian) [reply Packages/Status/Sources/Status/Row/StatusCardView.swift:67]: It's fine for card, card are the big link thing in some statuses. Basically it'll route in the app if it's a mastodon link and route to the correct browser if it's not. So it's fine to leave untouched there.

Comment (Dean151) [reply Packages/Status/Sources/Status/Row/StatusCardView.swift]: Let it be just a *very* small cleanup PR then

**Pull request merged**